.bazelrc

```diff
@@ -1,2 +1,2 @@
-test --test_tag_filters=nocov --test_output=all
+test --test_tag_filters=nocov --test_output=all --@io_bazel_rules_docker//transitions:enable=false
 test --spawn_strategy='processwrapper-sandbox'
```

Comment:
(My question should not block Code Review) It seems *the arguments that follow the first word are combined as if they had appeared on a single line*.  Then why repeating the word `test` 2 times is preferred? 

Is it semantic grouping? or is it because `.bazelrc` does not support line breaking ? :)

Reply:
I think it may just be a cosmetic choice to avoid overly long lines.